CHANGES/6399.bugfix

```diff
@@ -0,0 +1 @@
+Do not install C sources with binary distributions.
```

Comment:
I don't think it's a `bugfix`. Maybe it's a `misc` but definitely not something affecting the end-users' runtime.